### DISCUSSION TITLE
Can not find "cloudlog.sh"

### DISCUSSION THREAD

**Discussion by LarsThunberg** [magicbug/Cloudlog#1670] [General]
Hi!
According to the update 2.0 instructions:

`Theres some new cronjobs to add with version 2.0.
# Update the Cloudlog installation every day at midnight
0 0 * * * /bin/bash -c "Full-Path-To-Bash-Script/cloudlog.sh"`

But, I can not find this cloudlog.sh file in the Master branch.

//Lars SM0TGU

**Comment by magicbug** [comment]
Its relating to whatever bash script you call to update can change to reflect the directory

**Reply by LarsThunberg** [reply]
OK so what should be the content of "cloudlog.sh"? Is it the same as the current "update_cloudlog.sh" in the root folder?

**Reply by magicbug** [reply]
Yes
